.rubocop.yml

```diff
@@ -43,6 +43,9 @@ Style/IfInsideElse:
 Style/RedundantReturn:
   Enabled: false
 
+Style/WordArray:
+  MinSize: 5
```

Comment:
Sorry, I should have saved you the trouble I went through.

As for my choice of 5, though somewhat arbitrary... Mostly, I think `['string1', 'string2'].each do...` is more readable than `%w(string1 string2).each do ...`. Definitely `my_array = someCondition ? ['a', 'b'] : ['a']` is more readable than `my_array = someCondition ? %w(a b) : ['a']`. So I think we should be able to choose which seems more readable for short arrays (where readability may be context dependent) and enforce the style for long arrays.

Feel welcome (encouraged) to suggest another cutoff, especially looking at the set of changes increasing / decreasing the cutoff would allow / force.